pyproject.toml

```diff
@@ -70,7 +69,6 @@ module = [
 "trio/_core/_generated_io_windows",
```

Comment:
remove the ignores from pyproject.toml

Reply:
I think I have gotten requested removed in, 3e9c132, unresolve if more are required.